Add Binding Sets Reference

CommanderFoo opened the pull request:
First draft for the bindings reference.

Am unsure if the example (input type detection) with video should stay.

CommanderFoo committed:
Added Bindings Reference (Draft)

github-actions[bot] committed:
Optimised images with calibre/image-actions

CommanderFoo committed:
Info about no binding set will use default controls instead

CommanderFoo committed:
Passive voice

StanzillaManticore commented on src/references/binding_sets.md:
```diff
@@ -0,0 +1,266 @@
+
+For example, setting the key ++Q++ to perform the **Jump** action.
+
+1. Add **Basic Binding**
+2. Select **Jump** from the **Core Behavior** drop down
+3. Set the letter ++Q++ from the **Keyboard Primary** drop down
```
I think we always have periods at the end of list items

CommanderFoo replied:
Would you suggest that it is moved up, or not be an admonition and just be inline with the text?

StanzillaManticore replied:
I'm not sure actually, let's ask @slinkous as well.

StanzillaManticore reviewed (changes requested):
Very nice job! My only request is super minor and this looks great already. One thing that I'm not too sure about is the places where we explain something and then add the "BUT X" as a admonition afterwards. It does catch attention better but I feel like it kinda breaks the flow?

One example of this would be the "Deleting a Binding" section.

CommanderFoo committed:
Add periods for list

coreslinkous commented:
Agree that the info boxes should be merged into the text, and most of the warnings are too specific for a reference ... things like undo and sessions. I think the server context one is interesting, but it also seems like a really specific user error that isn't general enough to be called out here.

Also, cut all the properties/events/functions. Creators should go to the API for this.

coreslinkous commented on src/references/binding_sets.md:
```diff
@@ -0,0 +1,266 @@
+
+![!Core Behavior](../img/BindingSets/core_behaviors.png){: .center loading="lazy" }
+
+!!! info "Binding Defaults"
```
This, for example, is important enough that it should be in the main text. If this were a tutorial, the info box would be a perfect aside, but because it's central to the purpose of this doc I would keep it with the main text

coreslinkous commented on src/references/binding_sets.md:
```diff
@@ -0,0 +1,266 @@
+
+![!Networked Binding](../img/BindingSets/networked_bindings.png){: .center loading="lazy" }
+
+The following events and functions only work on the server if a binding is networked.
```
Change this to a link to the API entry

coreslinkous reviewed (changes requested):
See my comment on the PR

CommanderFoo committed:
Updated reference based on review

Removed warning / info boxes
Removed functions and events, linked to the API instead

StanzillaManticore committed:
Add link to forum post
